tests/jetty-testers/src/main/java/org/eclipse/jetty/tests/testers/JettyHomeTester.java

```diff
@@ -262,9 +262,9 @@ public static void unzip(Path archive, Path outputDir) throws IOException
                 }
             }
         }
-        catch (FileAlreadyExistsException | FileSystemAlreadyExistsException e)
```

Comment:
Why get rid of FileAlreadyExistsException?

Reply:
Haha I was thinking this as typo from the start :) 
but correct we need to catch this for  usage `Files.copy` 
restored